### ISSUE TITLE
SQLite ALTER not working

### ISSUE THREAD

**Issue by doublethink13** [sqlfluff/sqlfluff#6716]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

Running:

```bash
echo "ALTER TABLE pod_cidr DISABLE TRIGGER block_write_after_lock;" | sqlfluff lint --dialect sqlite -
```

Gives the following error:

```bash
== [stdin] FAIL
L:   1 | P:   1 |  PRS | Line 1, Position 1: Found unparsable section: 'ALTER
                       | TABLE pod_cidr DISABLE TRIGGER blo...'
WARNING: Parsing errors found and dialect is set to 'sqlite'. Have you configured your dialect correctly?
All Finished!
```

It seems this should have been fixed with [#5732](https://github.com/sqlfluff/sqlfluff/issues/5732)?

My sqlfuff version is `3.3.1`. (running as `docker run -i --rm -v $PWD:/sql sqlfluff/sqlfluff:3.3.1 "$@"`)

### Expected Behaviour

No errors.

### Observed Behaviour

`ALTER` seems to be unsupported.

### How to reproduce

```bash
echo "ALTER TABLE pod_cidr DISABLE TRIGGER block_write_after_lock;" | sqlfluff lint --dialect sqlite -
```

### Dialect

`sqlite`

### Version

`3.3.1` (running as `docker run -i --rm -v $PWD:/sql sqlfluff/sqlfluff:3.3.1 "$@"`)

### Configuration

None

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by doublethink13** [comment]
Is this because altering triggers in SQLite is not permitted?

**Comment by WittierDinosaur** [comment]
As far as I can tell this isn't valid sqlite syntax - per the docs https://www.sqlite.org/lang_altertable.html
